docs/AUTHENTICATION.md

```diff
@@ -0,0 +1,43 @@
+## Authentication
+Plio Analytics uses Auth0 authentication to manage access tokens and valid API requests from the client app, which in Plio's ecosystem is the frontend app.
+
+Plio Analytics relies on client credentials API which is suitable for machine-to-machine data exchange without any user consent. This generates access token for the client app instead of per-user access token.
+
+For more details on Auth0 configuration, check out the [official documentation from Cube.js](https://cube.dev/docs/security/jwt/auth0).
+
+### Configurating Auth0
+1. Sign up or log into your [Auth0 account](https://auth0.com/).
+2. From the left navigation, go to APIs and create a new API.
+   1. Name your API.
+   2. Set the identifier to your app url. Like `https://myapp.com/`. This must be a unique identifier and not to be reused in other APIs.
+   3. Signing Algorithm should be `RS256`.
+3. From the left navigation, go to Applications.
+   1. You will see an application was created automatically with your api name and `(Test application)`.
+   2. Select this app and go to settings tab. Rename by removing the (Test Application) part and save.
+4. Go to Quick Start tab and copy the example code for "Getting an access token for your API". Run the command in your shell. It should return a valid response with access token, expiry time and token type.
```

Comment:
what is to be done with the access token and other values? is this only for testing?

Reply:
This is only for testing. i'll update the step.